src/database/dbbuild.sql

```diff
@@ -1,53 +0,0 @@
-BEGIN;
```

Comment:
This file is missing?

Reply:
This file isn't unused, it will be used once we have to build the db

Reply:
There were two schema build files, one called dbSchema.sql which is used in the build script and the resetTestDb script, and one called dbBuild.sql which wasn’t used anywhere so I deleted it. dbSchema.sql is still there in /sql-files. I’m not sure how we ended up with two.